CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Release notes
 
 - Remove react-bootstrap from the GenomePositionSearchBox
+- Added the `nanAsZero` option for `horizontal-line` tracks
```

Comment:
Does this work for vertical-line track as well? And only line track or also bar tracks?

Reply:
It also works for vertical line tracks. The `horizontal` is a relic of the past that I used out of habit. `line` works as well. The vertical version uses the same code.

See here: https://github.com/higlass/higlass/blob/develop/app/scripts/TrackRenderer.js#L1581